frontend/talentsearch/src/js/components/experienceForm/ExperienceForm.tsx

```diff
@@ -147,19 +147,46 @@ export const ExperienceForm: React.FunctionComponent<ExperienceFormProps> = ({
 
   const labels = getExperienceFormLabels(intl, experienceType);
 
+  const pageTitle = () => {
+    switch (experienceType) {
+      case "award":
+        return intl.formatMessage({
+          defaultMessage: "Edit award",
+          id: "7X5cnb",
+          description: "Page title for the award profile form",
+        });
+      case "community":
+        return intl.formatMessage({
+          defaultMessage: "Edit community experience",
+          id: "LN0Wag",
+          description: "Page title for the community experience profile form",
+        });
+      case "education":
+        return intl.formatMessage({
+          defaultMessage: "Edit education experience",
+          id: "7DtNMY",
+          description: "Page title for the education experience profile form",
+        });
+      case "personal":
+        return intl.formatMessage({
+          defaultMessage: "Edit personal experience",
+          id: "96GHnx",
+          description: "Page title for the personal experience profile form",
+        });
+      case "work":
+        return intl.formatMessage({
+          defaultMessage: "Edit work experience",
+          id: "Ytlyzb",
+          description: "Page title for the work experience profile form",
+        });
+      default:
+        return "";
+    }
+  };
+
```

Comment:
Do we also want a case for if the user is adding a new one or editing an existing one? 😅